### ISSUE TITLE
Plans for new release?

### ISSUE THREAD

**Issue by gerroon** [SergiX44/XBackBone#245]
Hi SergiX44, 

Do you have any plans to make a release or preleae soon? I am very much interested in to test if the Slack fix works.

thanks

**Comment by yuvali1337** [comment]
You don’t have to wait for the next release for the Slack fix, you can add the bot name on your own for it to work it’s pretty simple.

**Comment by gerroon** [comment]
@yuvali1337 
Thanks for the hint. That did not seem to help :)

I am wondering if I need to setup something else to get this going. Does this need anything else on the Slcak side?

**Comment by SergiX44** [comment]
No, the fix is only server side, if you want to test it out tell me if it works.

**Comment by gerroon** [comment]
Changing that line did not help in Slack as far as I can tell. I still get no previews. Do need to post with special url?

>app/helpers.php and change 'Slackbot' to 'Slack'`

**Comment by yuvali1337** [comment]
Don't know whats the issue with Slack but i used it to fix Previews on Steam Chat and it worked but steam blocked my domain :(

**Comment by SergiX44** [comment]
I'll make some more testing with Slack before the next release.

**Comment by gerroon** [comment]
Did you also try this with the latest release or the unreleased master?
